wiki/Client/Beatmap_editor/Test_mode/de.md

```diff
@@ -0,0 +1,30 @@
+---
+tags:
+  - editor
+  - beatmapping
+  - mapping
+---
+
+# Testmodus
+
+![Screenshot des Testmodus](img/test-mode-example.jpg "Der Testmodus mit aktiviertem Autoplay")
+
+Der **Testmodus** ist eine Funktion des [Beatmap-Editors](/wiki/Client/Beatmap_editor) mit dem Ersteller ihre eigenen [Beatmaps](/wiki/Beatmap) zu Testzwecken aus dem Editor heraus spielen können. Der Modus ist über den Button `Test` in der rechten, unteren Ecke im Editor aufrufbar.
+
+![Testmodus Button](img/test-mode-button.png "Der Test-Button im Beatmap-Editor")
+
+*Anmerkung: Der Testmodus ist dafür bekannt, dass bei einigen Nutzern Lags oder spürbare Einbrüche der Framerate auftreten.*
```

Comment:
This sounds a bit like lagging is the main thing that the test mode is known for...